CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 2.5.0
+### Visual changes
+* Move "sync axis" and "show only main axis" settings to "Axis" card
+* Add font setttings for measure units and axis
+* Add an 'auto width' toggle for category labels that, when enabled, adjusts the label width to display all categories without trimming
+* Add settings to control axis display format and precision
+* Add settings to control bar size and to show percent of completion
+* Add settings to control the gap size between bars
+* Add "Legend" settings card
+* Update API to 5.11.0, tools to 5.6.0
```

Comment:
move this technical details to `code improvements` section

Reply:
Done.